packages/edit-post/src/components/welcome-guide/default.js

```diff
@@ -106,7 +106,7 @@ export default function WelcomeGuideDefault() {
 							</h1>
 							<p className="edit-post-welcome-guide__text">
 								{ __(
-									'New to the block editor? Want to learn more about using it? '
+									'New to the block editor? Want to learn more about using it?'
```

Comment:
Some of these may need to be tested manually, this one in particular stands out, as the text is followed by a link that could be on the same line.

The whole thing could be rewritten to use `createInterpolateElement`. 🤔